index.html

```diff
@@ -3478,27 +3446,25 @@ <h3><dfn>Writing content</dfn></h3>
             <ol>
               <li>
                 If |key| equals "`signal`" and |value| is not `undefined`, set
-                |reader|.<a>[[\Signal]]</a> to |value|.
+                |reader|.<a>[[\ScanAbortSignal]]</a> to |value|.
```

Comment:
How about doing the same as NDEFWriter and not use ScanAbortSignal, simply the signal from the dictionary member?

![image](https://user-images.githubusercontent.com/634478/99400706-92c6f180-28e7-11eb-94d7-1e2a3c9f212f.png)

Reply:
Should we then also remove the internal slot? (my pleasure)